[2022-12-03 14:51:40,285] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:18:00+00:00 [queued]>
[2022-12-03 14:51:41,370] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:18:00+00:00 [queued]>
[2022-12-03 14:51:41,371] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:51:41,371] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-03 14:51:41,428] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:51:43,362] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 18:18:00+00:00
[2022-12-03 14:51:43,978] {standard_task_runner.py:52} INFO - Started process 572 to run task
[2022-12-03 14:51:44,591] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T18:18:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpf6qrwylq', '--error-file', '/tmp/tmp1nw90u0w']
[2022-12-03 14:51:44,949] {standard_task_runner.py:80} INFO - Job 124: Subtask get_last_id
[2022-12-03 14:51:52,502] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:18:00+00:00 [running]> on host b41792e8e843
[2022-12-03 14:52:05,380] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T18:18:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T18:18:00+00:00
[2022-12-03 14:52:08,091] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-03 14:52:10,676] {python.py:173} INFO - Done. Returned value was: None
[2022-12-03 14:52:14,449] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T181800, start_date=20221203T145140, end_date=20221203T145214
[2022-12-03 14:52:16,361] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-03 14:52:19,451] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
